skyportal/handlers/api/candidate.py

```diff
@@ -384,6 +384,11 @@ def post(self):
         filters = Filter.query.filter(Filter.id.in_(filter_ids)).all()
         if not filters:
             return self.error("At least one valid filter ID must be provided.")
+
+        # use ra, dec as the discovery position unless it's already specified
+        obj.ra_dis = obj.ra if obj.ra_dis is None else obj.ra_dis
```

Comment:
This uses the first detection that passed a filter as the discovery coordinate. Are we sure we want to do that, rather than use the  first detection (discovery detection), as `*_dis`?

Reply:
We weren't doing anything before....
I'm assuming `discovery` is the moment something passes a filter, rather than the first time it's observed. So yes I think this correct.

Reply:
Ok, removing this in favor of moving the discovery position to Source/Candidate (another PR)